Platform/ElkhartlakeBoardPkg/Library/Stage2BoardInitLib/UpdateAcpiGnvs.c

```diff
@@ -49,6 +51,25 @@ typedef union {
   UINTN Data;
 } UINTN_STRUCT;
 
+/**
+  Check and update TSN controller device status
+
+  @param[in]  GbeBase          GBE PCI config space address
+**/
+VOID
+CheckGbeStatus (
+  UINTN GbeBase
+  )
+{
+  UINT16 PmeCtrlStatus;
+  //Check device status
+  PmeCtrlStatus = PciRead16 (GbeBase + PMECTRLSTATUS_OFFSET);
+  if (PmeCtrlStatus && POWERSTATE != 0) {

```

Comment:
`&& -> &`
What is the exact condition here? To check both bits together? or any bit in POWERSTATE?
```
if (PmeCtrlStatus && POWERSTATE != 0) {
if ((PmeCtrlStatus & POWERSTATE) != POWERSTATE) {
```

Reply:
Which condition is correct?